package.json

```diff
@@ -21,20 +21,19 @@
   "license": "AGPL-3.0",
   "dependencies": {
     "@babel/plugin-transform-runtime": "7.12.1",
-    "@edx/brand": "npm:@edx/brand-openedx@^1.1.0",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
```

Comment:
I'm assuming this pin is very deliberate?

Reply:
We pin everything now, I just saved the bot a PR, that's all.